### ISSUE TITLE
Tag 0

### ISSUE THREAD

**Issue by blcampbell** [NCRN/ForestVeg#79]
Determine why there is a tag 0 and if this value is needed for any reason.

![image](https://user-images.githubusercontent.com/36767405/46152713-9860a980-c23f-11e8-85b2-3c0c0ce89b66.png)

**Comment by johnpauls** [comment]
At a glance, it is seems like this is the result of a data entry error.

**Comment by lizmatthews03** [comment]
Yep, there is no information (no DBH, ID, location, etc) associated with this tag....and no note. Looks like a data entry error. Delete it.
